db/src/debug.rs

```diff
@@ -21,13 +21,20 @@ use rusqlite::types::{ToSql};
 use tabwriter::TabWriter;
 
 use bootstrap;
+use db::TypedSQLValue;
```

Comment:
I mean: you have `use mentat_core::…` below here, followed by `use types::Schema`. Rather than:

```
extern crate …;

use third_party_library::…;

use mentat_foo::…;
use mentat_bar::…;

use samecratemodule_a::…;
use samecratemodule_b::…;
```

you've interleaved them.